cfy_manager/components/prometheus/config/prometheus.yml

```diff
@@ -0,0 +1,56 @@
+# Prometheus self-check
+global:
+  scrape_interval:     15s
+  evaluation_interval: 30s
+  # scrape_timeout is set to the global default (10s).
+
+  external_labels:
+    monitor: cloudify
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:{{ prometheus.port }}']
+  - job_name: 'node'
+    static_configs:
+      - targets: ['localhost:{{ prometheus.node_exporter.metrics_port }}']
+  - job_name: 'blackbox_http_200'
+    metrics_path: /probe
+    params:
+      module: [http_200]  # Look for a HTTP 200 respose
+    static_configs:
+      - targets:
+          - http://{{ manager.private_ip }}:3000/  # Cloudify composer
+          - http://{{ manager.private_ip }}:8088/  # Cloudify stage
```

Comment:
if it's .private_ip, then :3000 and :8080 are not necessarily available, because those are not supposed to be exposed other than on localhost

it should be either private_ip and then `/composer` and `/` (stage), or 127.0.0.1 and the internal ports

Reply:
Added both